crossdock/endtoend/server/server.go

```diff
@@ -0,0 +1,91 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber/jaeger-client-go"
+	"github.com/uber/jaeger-client-go/config"
+)
+
+// Handler handles creating traces via jaeger-client.
+type Handler struct {
+	tracer opentracing.Tracer
+}
+
+type reporterConfig struct {
+	BufferFlushInterval int    `json:"bufferFlushInterval"`
+	LocalAgentHostPort  string `json:"localAgentHostPort"`
+}
+
+type initRequest struct {
+	Service  string                `json:"service"`
+	Sampler  *config.SamplerConfig `json:"sampler"`
+	Reporter *reporterConfig       `json:"reporter"`
+}
+
+type traceRequest struct {
+	Operation string            `json:"operation"`
+	Tags      map[string]string `json:"tags"`
+	Count     int               `json:"count"`
+}
+
+// Init initializes the jaeger client given the parameters in the request.
+func (h *Handler) Init(w http.ResponseWriter, r *http.Request) {
```

Comment:
do we really need this?  You are passing 3 parameters to initTracer:
  * service name - what's wrong with crossdock-go service name as in the other server?
  * agent's host-port - can't we just use a hardcoded name of the docker-compose service and a fixed port?
  * flush interval - why not just use 1s permanently?